src/routers/email.js

```diff
@@ -0,0 +1,12 @@
+const {Router} = require('express');
+const router = Router();
+const checkAuth = require('../middleware/auth');
+const checkRoleAuth = require('../middleware/role-auth');
+const {postEmail, postAccountEmail} = require('../controllers/email-controller');
+const {validationAccountEmail, validationSendEmail} = require('../validators/account-validator');
+
+router.post('/emails', validationSendEmail, postEmail); 
+
+router.post('/emails/account', checkAuth, checkRoleAuth(["manager"]), validationAccountEmail, postAccountEmail); 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
router.post("/emails/account", checkAuth, checkRoleAuth(["manager"]), validationAccountEmail, postAccountEmail); 
```